docs/core/diagnostics/sos-debugging-extension.md

```diff
@@ -0,0 +1,369 @@
+|    dumpmodule                       | Displays information about a EE module structure at the specified address.
+|    dumpmt                           | Displays information about a method table at the specified address.
+|    dumpobj                          | Displays info about an object at the specified address.
+|    dumpstack                        | Displays a native and managed stack trace.
+|    eeheap                           | Displays info about process memory consumed by internal runtime data structures.
+|    eestack                          | Runs dumpstack on all threads in the process.
+|    gcroot                           | Displays info about references (or roots) to an object at the specified address.
+|    histclear                        | Releases any resources used by the family of Hist commands.
+|    histinit                         | Initializes the SOS structures from the stress log saved in the debuggee.
+|    histobj                          | Examines all stress log relocation records and displays the chain of garbage collection relocations that may have led to the address passed in as an argument.
+|    histobjfind                      | Displays all the log entries that reference an object at the specified address.
+|    histroot                         | Displays information related to both promotions and relocations of the specified root.
+|    ip2md                            | Displays the MethodDesc structure at the specified address in code that has been JIT-compiled.
+|    loadsymbols                      | Load the .NET Core native module symbols.
+|    name2ee                          | Displays the MethodTable structure and EEClass structure for the specified type or method in the specified module.
+|    pe                               | Displays and formats fields of any object derived from the Exception class at the specified address.
+|    setclrpath                       | Set the path to load coreclr dac/dbi files. setclrpath `<path`>
+|    sethostruntime                   | Sets or displays the .NET Core runtime directory to use to run managed code in SOS.
+|    setsymbolserver                  | Enables the symbol server support.
+|    setsostid                        | Set the current os tid/thread index instead of using the one lldb provides. setsostid `<tid`> `<index`>
+|    sos                              | Various coreclr debugging commands. See 'soshelp' for more details. sos `<command-name`> `<args`>
+|    soshelp                          | Displays all available commands when no parameter is specified, or displays detailed help information about the specified command. soshelp `<command`>
+|    syncblk                          | Displays the SyncBlock holder info.
+
+## Examples
+
+### Windbg/cdb Example Usage
+
+The following command displays the contents of an array at the address `00ad28d0`.  The display starts from the second element and continues for five elements.
```

Comment:
I think these examples would be better in a table layout.

| Command | Description |
| - | - |
| `!dumparray -start 2 -length 5 -detail 00ad28d0` | Displays the contents of an array at the address `00ad28d0`.  The display starts from the second element and continues for five elements. |
| `!dumpassembly 1ca248` | Displays the contents of an assembly at the address `1ca248`. |

etc. It's hard to read as-is.

Reply:
Thanks! I have changed it to a table format.